src/Mod/ModScript.h

```diff
@@ -313,6 +321,7 @@ class ModScript
 		ReactionUnitAction,
 		ReactionUnitReaction,
 
+		PsiAttack,
```

Comment:
is this only for the new battle type? or also for normal psi amps? if also for normal psi amps, then please split it into a separate commit... don't put multiple new features into the same commit please

Reply:
Will do